package.json

```
@@ -44,7 +44,9 @@
     "nodeDebug": "npm run buildDebug && npm run launch",
     "bun": "npm run build && bun dist/node/index.mjs",
     "cfDev": "npm run build && wrangler dev",
-    "cfDeploy": "npm run build && wrangler deploy"
+    "cfDeploy": "npm run build && wrangler deploy",
+    "prettier:fmt:check": "prettier -c . '!./tmp' --log-level warn",
+    "prettier:fmt:fix": "prettier -w . '!./tmp' --log-level warn"
```

Wdyt of this?

1. `fmt` over `prettier:fmt` - IMO script name shouldn't have "implementation details"
2. `:check` can be implicit
3. `!./tmp` isn't necessary here
4. Let's add `--log-level warn` if we ever realize it's needed

```suggestion
    "fmt": "prettier -c .",
    "fmt:fix": "prettier -w ."
```